livekit-agents/livekit/agents/job.py

```diff
@@ -44,8 +43,8 @@ class JobAcceptArguments:
 
 @dataclass
 class RunningJobInfo:
+    accept_arguments: JobAcceptArguments
```

Comment:
what's the thinking in making these longer?

Reply:
this was public but not exposed to the users